client/src/components/shared/Layout/LayoutWallet/LayoutWallet.tsx

```diff
@@ -84,7 +87,15 @@ const LayoutWallet: FC<LayoutWalletProps> = ({ onDisconnect }) => {
         title={t('wallet')}
         titleSuffix={
           address ? (
-            <div className={styles.titleSuffix}>{truncateEthAddress(address)}</div>
+            <Button
+              className={styles.address}
+              dataTest="LayoutWallet__Button--address"
+              href={`${networkConfig.etherscanAddress}/address/${address}`}
+              variant="link3"
```

Comment:
`link` and `link5` variants already have `arrowTopRight` declared inside. They can't be reused?

Reply:
Nice catch -> changed to `link`